CMakeLists.txt

```diff
@@ -147,13 +147,13 @@ ELSE()
         SET(CMAKE_HIP_ARCHITECTURES ${Kokkos_HIP_ARCHITECTURES})
       ENDIF()
     ENDIF()
-    IF(${Kokkos_VERSION} VERSION_GREATER_EQUAL "4.5.01")
+    IF(${Kokkos_VERSION} VERSION_GREATER_EQUAL "4.6.01")
```

Comment:
```suggestion
    IF(${Kokkos_VERSION} VERSION_GREATER_EQUAL "4.6.02")
```

Reply:
@lucbv I updated with signoff in e9710dfc65d09f5214fe6474315e52bf0f6cc8b0